docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -0,0 +1,227 @@
+# eZ Platform v3.0 deprecations and BC breaks
+
+This page lists backwards compatibility breaks and deprecations introduced in eZ Platform v3.0.
+
+## Backwards compatibility breaks
+
+### Symfony 4
+
+v3.0 now uses Symfony 4 instead of Symfony 3.
+Refer to [Symfony changelog](https://github.com/symfony/symfony/blob/master/CHANGELOG-4.0.md)
+and [Symfony upgrade guides](https://github.com/symfony/symfony/blob/master/UPGRADE-4.0.md)
+to learn about all changes it entails.
+
+See [v3.0 project update](ez_platform_v3.0_project_update.md) for the steps you need to take to update your project to Symfony 4.
```

Comment:
We can also add a link to [Requirements](../getting_started/requirements.md) it would fit nicely in this context.